CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(COMPONENT_ADD_INCLUDEDIRS
 
 set(COMPONENT_PRIV_INCLUDEDIRS "src")
 set(COMPONENT_SRCEXCLUDE "src/mdnsresponder.c")
-set(COMPONENT_REQUIRES wolfssl cJSON http-parser mdns)
+set(COMPONENT_REQUIRES wolfssl cJSON http-parser mdns spi_flash)
```

Comment:
the component.mk file was changed to use the embedded json parser in the ESP IDF. does it make sense to update it here also?